### ISSUE TITLE
Live version of the website

### ISSUE THREAD

**Issue by drex44** [sudz123/Happy_Hacking#179]
I couldn't find the live version of the website hosted anywhere.
It would be awesome to check the latest code somewhere live.

so it can be hosted somewhere on notify, surge or similar providers. they don't charge for static websites!

**Comment by sudz123** [comment]
Yup I agree, I was thinking of starting a gh-pages branch and hosting it on github. Will do it in a day's time. If you have any other better suggestions do let me know.

**Comment by Apoorva-13** [comment]
can i work on this , i am new to this . i know how to deploy  a static website on heroku but would need help to understand gh-pages or netlify?

**Comment by sudz123** [comment]
Don't take up this issue. This one is for the maintainers. You will require a higher level of access to the code for this.

**Comment by drex44** [comment]
@sudz123 using netlify is pretty easy and will take only 10 minutes to set up. I have done it in my repos. It automatically deploys whenever new commit comes. I think you should do this one as soon as possible. it will be a big help for new contributors and will give motivation to existing contributors as they can see their work live!

P.S. I got inspired with your repo. Nice job.

**Comment by sudz123** [comment]
Thanks @drex44 , the whole idea is to bring up new programmers to the open source community and get them started. I will be doing this, but when I do, I will be sharing this in a much broader community to show how this was achieved. I can only share this when I am satisfied with the site. Waiting on #212 , the moment this is closed, I will make the site go live.

**Comment by sudz123** [comment]
I am having trouble using both gh-pages and netlify since our site is not to be built from the root directory. Do you know how this is done? We have multiple static pages to be hosted.

**Comment by drex44** [comment]
I haven't set up local dev env for this repo so give sometime. I will check it out and update you on how to configure it.

**Comment by drex44** [comment]
@sudz123 you have to keep the index.html file in the root of the repo or docs/ folder. currently, you have home.html. you have to rename it. then you can use gh-pages.
I tried and tested it. it works just fine.

**Comment by sudz123** [comment]
@drex44 This is one way. I am also trying to find if it is possible to keep the same structure and build the site without renaming anything. Maybe add a script in root which will redirect to home.html in src. If this is not possible I will do what you are suggesting. Do you know of any such functionality which github provides? Otherwise I will have to hard code a script maybe.

**Comment by drex44** [comment]
if you don't want to change anything then try using heroku with express.js to serve static sites. this will require you to add server.js file and forward requests coming to home.html from index.html.
